roles/st2repos/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# tasks file for st2repos
+
+- name: Install prereqs
+  sudo: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+    update_cache: yes
+  with_items:
+    - debian-archive-keyring
+    - apt-transport-https
+  
+- name: Add keys to keyring
+  sudo: yes
+  apt_key:
+    url: https://packagecloud.io/StackStorm/stable/gpgkey
+    state: present
+
+- name: Add StackStorm repos
+  sudo: yes
+  apt_repository:
+    repo: 'deb https://packagecloud.io/StackStorm/stable/ubuntu/ trusty main'
+    state: present
+    update_cache: yes
+
+- name: Add StackStorm src repos
```

Comment:
`deb-src` is not needed, since we don't have repo with deb sources.

So this block is good to avoid/delete.